src/components/Settings/SettingsMain.tsx

```diff
@@ -219,6 +226,28 @@ const SettingsMain: React.FC = () => {
                     )}
                   </div>
                 </div>
+                <div className="form-row">
```

Comment:
This option/row should also probably be immediately above/below the "Hide Available Media" option.  We should try to group/order the options in some logical way, and IMO the application URL, proxy, and CSRF settings should be together.